.github/workflows/github-actions-test-installer.yml

```diff
@@ -2,14 +2,16 @@ name: Test DependencyInstaller script
 on:
   schedule:
   - cron: "0 8 * * MON"
+  push:
+  pull_request:
```

Comment:
Remove as we don't want to trigger this on every push/PR. I suggested on other PR to trigger only when the relevant files change.